pkg/server/k8s/client_test.go

```diff
@@ -374,7 +374,31 @@ func TestHasAnotherIngress(t *testing.T) {
 		t.Fatal("got unexpected error:", err)
 	}
 
-	if rs != false {
+	if rs != true {
 		t.Errorf("got true; want false")
 	}
 }
+
+func TestNginxAnnotations(t *testing.T) {
+	cli := &Client{testing: true}
+	namespace := "test"
+	ingName := "ingress1"
+
+	if err := cli.createIngress(namespace, ingName, []string{"xpto.com"}, "nginx"); err != nil {
+		t.Fatal("got unexpected error:", err)
+	}
+
+	ing, err := cli.fake.ExtensionsV1beta1().Ingresses(namespace).Get(ingName, metav1.GetOptions{})
+	if err != nil {
+		t.Fatal("got unexpected error:", err)
+	}
+	t.Logf("ingress annotations %v\n", ing.Annotations)
```

Comment:
Is it necessary?

Reply:
I'll keep this here while the PR is a draft, as these tests are still failing. For some reason, the Annotations never return, even if we set them :/